src/index.html

```diff
@@ -11,8 +14,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__block">
+        <img
+          src="./images/imac.jpeg"
+          alt="product card image"
+          class="product-card__image"
+        />
+      </div>
+
+      <div class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__product-code">Product code: 195434</div>
+
+      <div class="product-card__reviews">
+        <div class="stars stars--4">
```

Comment:
The 'stars' block should be a separate BEM block and not an element of 'product-card'. This allows it to be reused independently of the 'product-card'.